web-registry/src/__snapshots__/storyshots.test.ts.snap

```diff
@@ -3420,6 +3420,36 @@ exports[`Storyshots Tooltip Info 1`] = `
 </MemoryRouter>
 `;
 
+exports[`Storyshots Tooltip Info With Icon 1`] = `
```

Comment:
@victordelval is this how we get this tool tip info icon into storybook? so we can preview how it looks?

Reply:
actually these are the snapshots (visual testing) of the components that we make from the registry app. 

To display the component in the storybook, the component was added in this other type of "stories" files: `web-components/src/components/tooltip/tooltip.stories.tsx`